src/test/java/sorinfoca/Tests/Tema8/TestPage/TestPages.java

```diff
@@ -0,0 +1,103 @@
+package sorinfoca.Tests.Tema8.TestPage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestPages {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    private ChromeOptions mobileEmulation() {
```

Comment:
This configuration will be put inside BaseTest in @BeforeMethod for mobile group without any return type